packages/rescript-mui-material/src/components/InputBase.res

```diff
@@ -76,8 +76,15 @@ type inputMode =
   | @as("email") Email
   | @as("url") Url
 
-type inputBaseComponentProps = {
-  ...CommonProps.clickableProps,
+/**
+ https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea#wrap
+ */
+type wrap =
+  | @as("hard") Hard
+  | @as("soft") Soft
+  | @as("off") Off
+
+type inputBaseComponentPublicProps = {
```

Comment:
IMO, this should probably be in `CommonProps` for inputs / textarea props

Reply:
Yes, please move them.